src/Tribe/API.php

```diff
@@ -159,8 +191,12 @@ public static function saveEventMeta( $event_id, $data, $event = null ) {
 				$data['EventCost'] = reset( $data['EventCost'] );
 			}
 
-			if ( isset( $data['FeaturedImage'] ) && ! empty( $data['FeaturedImage'] ) ) {
-				update_metadata( 'post', $event_id, '_thumbnail_id', $data['FeaturedImage'] );
+			if ( isset( $data['FeaturedImage'] ) ) {
```

Comment:
Maybe we can use the array utils here?

Reply:
I don't think we might be able to do it in this case as we need to check:

1. if the key exists on the array.
1.1 If that's the case then check if the value on that key is not empty which can be `false` or `null` in that case delete the meta
1.2 If is not `empty` add the meta to the post.

However I'm not sure how the `Tribe__Utils__Array` might fit in this use case.

Thanks.